lib/pages/settings_emotes/settings_emotes_view.dart

```diff
@@ -23,6 +25,31 @@ class EmotesSettingsView extends StatelessWidget {
       appBar: AppBar(
         leading: const BackButton(),
         title: Text(L10n.of(context)!.emoteSettings),
+        actions: [
```

Comment:
I honestly prefer the prominent buttons but I can understand it's not anyone's preference. LGTM hence apart from the comment above.